state/state.go

```diff
@@ -2060,11 +2060,15 @@ func (st *State) SetAdminMongoPassword(password string) error {
 }
 
 type controllersDoc struct {
-	Id              string `bson:"_id"`
-	CloudName       string `bson:"cloud"`
-	ModelUUID       string `bson:"model-uuid"`
-	MachineIds      []string
-	MongoSpaceName  string `bson:"mongo-space-name"`
+	Id         string `bson:"_id"`
+	CloudName  string `bson:"cloud"`
+	ModelUUID  string `bson:"model-uuid"`
+	MachineIds []string
+
+	// These are replaced by juju-ha-space in controller config.
+	// Deprecated.
+	MongoSpaceName string `bson:"mongo-space-name"`
+	// Deprecated.
 	MongoSpaceState string `bson:"mongo-space-state"`
 }
```

Comment:
The other upgrade steps have taken to removing them entirely from the 'active' document definition in state/state.go, and instead the upgrade step itself has a copy of the old doc type and uses that for serialization.
That way our current docs don't have a lot of deprecated items in them just to upgrade off of them.

Reply:
Ack; will change.